src/test/java/com/github/pires/obd/commands/SpacesOffCommandTest.java

```diff
@@ -0,0 +1,17 @@
+package com.github.pires.obd.commands.protocol;
+
+import junit.framework.TestCase;
+
+/**
+ * Created by sjanvan1 on 2015/09/15.
+ */
+public class SpacesOffCommandTest extends TestCase {
```

Comment:
This doesn't follow the structure of this project unit-tests.

See https://github.com/pires/obd-java-api/blob/master/src/test/java/com/github/pires/obd/commands/RPMCommandTest.java